src/board.py

```diff
@@ -85,9 +85,12 @@
 elif board_id == ap_board.CORAL_EDGE_TPU_DEV:
     from adafruit_blinka.board.coral_edge_tpu import *
 
-elif board_id == ap_board.ODROID_C2:
-    from adafruit_blinka.board.odroidc2 import *
+#elif board_id == ap_board.ODROID_C2:
+    #from adafruit_blinka.board.odroidc2 import *
 
```

Comment:
Is there a reason this was commented out?

Reply:
I meant it seems like commenting this out would break it for the Odroid C2 board.